docs/README.md

```diff
@@ -19,9 +19,24 @@ The [Amazon Elastic File System](https://aws.amazon.com/efs/) Container Storage
 Currently only static provisioning is supported. This means an AWS EFS file system needs to be created manually on AWS first. After that it can be mounted inside a container as a volume using the driver.
 
 The following CSI interfaces are implemented:
+* Controller Service: CreateVolume, DeleteVolume, ControllerGetCapabilities, ValidateVolumeCapabilities
```

Comment:
I think there should be a (temporary) note that you need a prerelase version of the driver, because I'm planning to release 1.1.0 with the bottlerocket fix and various version bumps, and then do the dynamic provisioning release in 1.2.0 or something.

Reply:
What will be the pre-release version?

Reply:
Hmm since it doesn't exist yet, I think we can hold this PR until it does, then we will be able to provide clearer instructions for how to try it